envi/qt/memcanvas.py

```diff
@@ -29,14 +29,12 @@ def javaScriptConsoleMessage(self, level, msg, line, source):
 
 class VQMemoryCanvas(e_memcanvas.MemoryCanvas, QWebEngineView):
 
-    #syncSignal = QtCore.pyqtSignal()
     def __init__(self, mem, syms=None, parent=None, **kwargs):
         e_memcanvas.MemoryCanvas.__init__(self, mem=mem, syms=syms)
         QWebEngineView.__init__(self, parent=parent, **kwargs)
 
         self._canv_cache = None
         self._canv_curva = None
-        self._canv_rendtagid = '#memcanvas'
```

Comment:
i assume this has something to do with it being explicitly created somewhere?  or is this because you're explicitly setting `sel` to `'#memcanvas'` if we don't hand in something else?

Reply:
thanks.  i did see that, but forgot to remove this comment.